src/sentry/snuba/metrics/query_builder.py

```diff
@@ -289,33 +285,15 @@ def _parse_orderby(self, query_params):
 
         return MetricsOrderBy(field, direction)
 
-    def _parse_limit(self, paginator_kwargs):
-        limit = paginator_kwargs.get("limit")
-        return Limit(limit) if limit else None
-
-    def _parse_offset(self, paginator_kwargs):
-        offset = paginator_kwargs.get("offset")
-        return Offset(offset) if offset else None
-
-    def _validate_series_limit(self, query_params):
-        if self.limit and query_params.get("per_page") is not None:
-            if (
-                self.end - self.start
-            ).total_seconds() / self.rollup * self.limit.limit > MAX_POINTS:
-                raise InvalidParams(
-                    f"Requested interval of {query_params.get('interval', '1h')} with statsPeriod of "
-                    f"{self.end-self.start} is too granular for a per_page of "
-                    f"{self.limit.limit} elements. Increase your interval, decrease your statsPeriod, "
-                    f"or decrease your per_page parameter."
-                )
-
+    def _parse_limit(self, query_params, paginator_kwargs):
```

Comment:
Now that we have this check in the endpoint: https://github.com/getsentry/sentry/pull/34287/files/db9a9f6797dd57b9ff5bb3070a6a6aad8667016f..4d9cbfe6e8914afc93e0fcdba7335b4d4c8b364f#diff-9710b6465f5f9c1c56f631d8fe1ce6e91fd9aa3e557ad788402fd5ef33d866f6R26

Should we just remove `_parse_limit` and `_parse_offset`?

Reply:
I would say not now because the metrics API is still used. maybe when we remove it

Reply:
Right, my bad, I confused the sessions endpoint with the metrics endpoint.